documentation/modules/configuring/con-exposing-kafka-connect-api.adoc

```diff
@@ -26,12 +26,11 @@ curl -X POST \
   -d '{ "name": "my-source-connector",
     "config":
     {
-      "class":"org.apache.kafka.connect.file.FileStreamSourceConnector",
+      "connector.class":"org.apache.kafka.connect.file.FileStreamSourceConnector",
       "file": "/opt/kafka/LICENSE",
       "topic":"my-topic",
       "tasksMax": "4",
-      # ...
-    }
+      "type": "source"}
```

Comment:
Keeping `}` on a separate line as it was will make it much more readable. It will also follow the way the opening `{` is formatted.